pfio/cache/file_cache.py

```diff
@@ -80,7 +80,7 @@ class FileCache(cache.Cache):
     '''Cache system with local filesystem
 
     Stores cache data in local temporary files, created in
-    ``~/.chainer/chainerio/cache`` by default. Cache data is
+    ``~/.chainer/pfio/cache`` by default. Cache data is
```

Comment:
Maybe it is better to put under `.pfn/`?

Reply:
How about `~/.pfio/cache` ?